src/lib/utilities/update-query-parameters.ts

```diff
@@ -35,6 +35,10 @@ export const updateQueryParameters = async ({
   }
 
   if (browser && url.href !== window.location.href) {
+    // Needed to trigger store updates when used as npm package
+    if (!url.href.endsWith('#')) {
```

Comment:
This is gross, but this works well. It doesn't actually add a '#' in the url due to Svelte magic.

Reply:
I hate it but I trust you.